.github/workflows/ci.yaml

```diff
@@ -54,11 +54,19 @@ jobs:
       matrix:
        include:
          - helm-version: v3.4.2
+           extra-helmfile-flags: ""
          - helm-version: v3.5.4
+           extra-helmfile-flags: ""
          - helm-version: v3.6.3
+           extra-helmfile-flags: ""
          - helm-version: v3.7.2
+           extra-helmfile-flags: ""
          - helm-version: v3.8.2
+           extra-helmfile-flags: ""
          - helm-version: v3.9.0
+           extra-helmfile-flags: ""
+         - helm-version: v3.9.0
+           extra-helmfile-flags: "--enable-live-output"
```

Comment:
@yxxhero To avoid testing every combinations of these parameters which might make the ci unnecessarily slow, probably we'd better revert https://github.com/helmfile/helmfile/pull/361 and use this `include` syntax to select a few meaning combinations of these two settings along with the recently added helm-secrets plugin version (https://github.com/helmfile/helmfile/pull/360)?

Also, I got to know the helm project only maintains the latest minor version of helm (https://helm.sh/docs/topics/version_skew/#supported-versions) at a time so probably we can safely remove 3.4.2 to 3.8.2 out of the matrix.

The end result would look like this:

```
       include:
         - helm-version: v3.9.4
           plugin-secrets-version: 3.15.0
           extra-helmfile-flags:
         - helm-version: v3.9.4
           plugin-secrets-version: 4.0.0
           extra-helmfile-flags:
         - helm-version: v3.9.4
           plugin-secrets-version: 4.0.0
           extra-helmfile-flags: "--enable-live-output"
```

WDYT?

Reply:
@mumoshu I agree with you. I will create a PR for this.

Reply:
@mumoshu I see you already do this. Good job. Thanks very much.